lib/community/pages/community_page.dart

```diff
@@ -28,6 +29,22 @@ class _CommunityPageState extends State<CommunityPage> with AutomaticKeepAliveCl
   @override
   bool get wantKeepAlive => true;
 
+  final GlobalKey<NestedScrollViewState> globalKey = GlobalKey();
+
+  @override
+  void initState() {
+    super.initState();
+    WidgetsBinding.instance!.addPostFrameCallback((timeStamp) {
+      globalKey.currentState!.innerController.addListener(() {
+        if (globalKey.currentState!.innerController.position.pixels >= globalKey.currentState!.innerController.position.maxScrollExtent * 0.7) {
+          print('endScroll');
+          // How do we call this from here?
+          // context.read<CommunityBloc>().add(GetCommunityPostsEvent(communityId: widget.communityId));
```

Comment:
Unfortunately, we won't be able to call this function in the initState function since it has no access to the context.

One "hacky" way to pass in the context is to allow CommunityPage to accept a BuildContext as one of its parameters and then use that context here. However, I'm not sure if that's the best way to go about it